lib/membrane_flv_plugin/format.ex

```diff
@@ -41,6 +41,7 @@ defmodule Membrane.FLV do
   def index_to_sound_format(11), do: :Speex
   def index_to_sound_format(14), do: :MP3_8k
   def index_to_sound_format(15), do: :device_specific
+  def index_to_sound_format(_index), do: {:error, "Unknown audio index"}
```

Comment:
I meant, raise an error
```suggestion
  def index_to_sound_format(_index), do: raise "Unknown audio index #{inspect(index)}"
```